### DISCUSSION TITLE
Github Student developer pack

### DISCUSSION THREAD

**Discussion by Brandon-Isaac** [community/community#123432] [GitHub Education]
## Why does it take this long for the pack to be approved

**Comment by Davdadev** [comment]
it should not only take a few days if you need more help go to https://support.github.com/

**Comment by ghost** [comment]
## Answer 🌟

- GitHub Student Verification take few days time (around 4-7) days to get verified.

- GitHub check for the student/teacher identity by --
    - Verifying there actual name from College / Educational Record.
    - Verifying there institutional mail id.
    - Verifying from the IP Addess you are applying.
    - Your Photo Id Verification ``` Institutional Id Card with your and Institional Full Name```
    - Your GitHub Profile belong to Human not to any bot ``` By your Profile Photo and Profile ReadMe```.
    - Your Billing details.
  
All this and more process behind the scene take place and take manual human involvement that why it take sometime .

> Good Luck 😊
